pop-api/integration-tests/src/lib.rs

```diff
@@ -3,39 +3,40 @@
 use frame_support::{
 	assert_ok,
 	traits::fungibles::{
-		approvals::Inspect as ApprovalInspect, metadata::Inspect as MetadataInspect, Inspect,
+		approvals::Inspect as ApprovalInspect, metadata::Inspect as MetadataInspect,
```

Comment:
FYI - if trait just needs to be in scope, and not explicitly used, can just use `as _` as a simpler approach which avoids naming clashes.